### ISSUE TITLE
Stream title is not displayed

### ISSUE THREAD

**Issue by edio** [arybczak/ncmpcpp#44]
On my working machine, `Ubuntu`, `ncmpcpp 0.5.10`, `mpd 0.18.7` when playing this stream http://xstream1.somafm.com:6200 ncmpcpp displays following string as filename 

````
Def Con Radio: SomaFM's special mix for Def Con [SomaFM]
````
However, on my home machine `Archlinux`, `ncmpcpp 0.6.4`, `mpd 0.19.9` it displays just url as filename.

Meanwhile, mpc shows the following:

````
Def Con Radio: SomaFM's special mix for Def Con [SomaFM]: Different Marks - You Help Me (feat. Nnaji)
[playing] #3/3  22:56/0:00 (0%)
volume: 41%   repeat: off   random: off   single: off   consume: off
````

Thus I guess mpd provides all required info and ncmpcpp is one to blame.
Reproduced on the latest build from git.

**Comment by arybczak** [comment]
Works for me on either 0.6.4 and master (url turns into "artist - title" after a second after stream starts).

**Comment by edio** [comment]
The problem is not song title, but stream title, which is normally displayed as filename on Song info screen.
http://imgur.com/Jv9XmV9

Unfortunately, I can't provide screenshot from my working machine now, but the title for this stream should be 
````
Def Con Radio: SomaFM's special mix for Def Con [SomaFM]
````
and for other streams you can check it by calling mpc without arguments. Stream title will be prepended to song title.

**Comment by edio** [comment]
I built 0.5.10 from sources, and made a screenshot. So can confirm bug on the same machine
http://imgur.com/hYAQQWi

**Comment by edio** [comment]
I tried to do git bisect. Unfortunately, certain commits were not buildable at all, and I just marked them as `bad` during bisect procedure.
So I don't know bad commit name exactly, but here is full bisect log. Hope it might be of any help.

````
# bad: [2ea7db1d20f919f7691eef61118c0574bf0518f1] man: update the description of window_border_color
# good: [6039330c6983bbe7cd7b2f19f872c924cf310567] update NEWS / 0.5.10 release
git bisect start '0.6.3' '0.5.10'
# bad: [6e33c2ef0f103dc7cb77b76bc98f7d4721722184] media library: preserve filters/selections on update
git bisect bad 6e33c2ef0f103dc7cb77b76bc98f7d4721722184
# bad: [fe9bcfbeab641307d9ea58f9cc64c428d0e9986d] bindings: add support for parsing/displaying insert key
git bisect bad fe9bcfbeab641307d9ea58f9cc64c428d0e9986d
# bad: [0447b4e881e537a4984d4c15c7ff9efc2923f4b1] window: remove GetXY
git bisect bad 0447b4e881e537a4984d4c15c7ff9efc2923f4b1
# good: [fbd4a0484ecbcb754406106bd849096361186586] fix a few issues with win32 build
git bisect good fbd4a0484ecbcb754406106bd849096361186586
# bad: [397783d7012b69a86cdb0d3009d25fab065ccff3] reduce usage of pointers / replace std::pair in SearchEngine
git bisect bad 397783d7012b69a86cdb0d3009d25fab065ccff3
# good: [08adcbfdc87be5f5c10c716969618589a9d98181] screen: make behavior of locked screens more sensible in a certain case
git bisect good 08adcbfdc87be5f5c10c716969618589a9d98181
# bad: [2c50064ba5404f60e79666bc396b364334ac18c8] playlist: move sorting functions to Playlist::EnterPressed
git bisect bad 2c50064ba5404f60e79666bc396b364334ac18c8
# good: [9e071b074c2511e46920717d5a041ea39ac8d9f9] display: rewrite Display::Columns and Display::SongsInColumns
git bisect good 9e071b074c2511e46920717d5a041ea39ac8d9f9
# good: [e0d5daef4c2c7c7c38b18c77d1cf647181fde589] fix {Clock,Visualizer}::Scroll so they override base function
git bisect good e0d5daef4c2c7c7c38b18c77d1cf647181fde589
# first bad commit: [2c50064ba5404f60e79666bc396b364334ac18c8] playlist: move sorting functions to Playlist::EnterPressed
````

**Comment by arybczak** [comment]
Fixed in 0.6.x.
